### DISCUSSION TITLE
Display Macro, Caselight Macro

### DISCUSSION THREAD

**Discussion by 330Speedy** [QIDITECH/QIDI_Q1_Pro#107] [Q&A] [answered]
Hi, I want to ask you for advice
Where can I find a macro (if there is one) that controls the automatic display shutdown? (screensaver)
I would like to set the printer's caselight to automatically turn off together with the display. (I think it was even in some of the FW versions)
I usually print long prints for which it is unnecessary for the lighting to be on, and it is enough for it to turn off again after a while during a routine check.

Thank you very much to everyone in advance for your advice.

**Comment by QIDITECH** [comment, marked as answer]
Sorry, the extinguishing function is controlled by the screen itself, no macros are used.

**Reply by 330Speedy** [reply]
Thank you for your reply. 
So I ask, is there any way to set this up? I think it's about modifying the config file for the screen? Is there any easy way or will other features break! 
Thank you.

**Reply by QIDITECH** [reply]
I'm sorry. There's no better way.
You can only control the LED light on the screen, and then set the automatic screen time.
